wagtail/wagtailadmin/static_src/wagtailadmin/scss/components/_datetimepicker.scss

```diff
@@ -298,7 +299,9 @@
 }
 
 .xdsoft_noselect::selection { background: transparent; }
+
 .xdsoft_noselect::-moz-selection { background: transparent; }
+
 .xdsoft_datetimepicker.xdsoft_inline {
```

Comment:
That "newline between rulesets" reformatting looks useful. The one between nested statements or after inline comments does not.